incubator/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/config/MqttBindingConfig.java

```diff
@@ -39,13 +38,22 @@ public MqttBindingConfig(
         this.routes = binding.routes.stream().map(MqttRouteConfig::new).collect(toList());
     }
 
+    //TODO: delete?
+    public MqttRouteConfig resolve(
+        long authorization)
+    {
+        return routes.stream()
+            .filter(r -> r.authorized(authorization))
+            .findFirst()
+            .orElse(null);
+    }
+
```

Comment:
Do we need an ability to now route by `PUBLISH (topic)`, `SUBSCRIBE (topic)` and `SESSION` so that we can tell where to send the `MqttPublishBeginEx` streams vs the `MqttSubscribeBeginEx` streams vs the `MqttSessionBeginEx` stream?

Note: we no longer have a synthetic `mqtt` topic name for the session and have a `MqttSessionBeginEx` stream instead.